.github/workflows/go.yml

```diff
@@ -2,9 +2,13 @@ name: Go Tests
 
 on:
   push:
-    branches: [ main ]
+    branches:
+      - main
+      - 'feat-**'
```

Comment:
Should this just be `feat-*` (one *)?

Reply:
in github syntax, `**` to matches any character. `*` matches any character except `/`.
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

 im fine with either, i kind of just copied one of the examples in the github documentation page which used `**`